packages/editor/src/core/all-plugins.tsx

```diff
@@ -0,0 +1,576 @@
+    icon: IconGeogebra,
+    type: EditorPluginType.Geogebra,
+    initialState: {
+      plugin: 'rows',
+      state: [{ plugin: 'geogebra', state: '' }],
+    },
+  },
+  {
+    ...getInternationalizedPluginStrings(EditorPluginType.H5p),
+    icon: IconH5p,
+    type: EditorPluginType.H5p,
+    initialState: {
+      plugin: 'rows',
+      state: [
+        {
+          plugin: 'exercise',
+          state: {
+            content: {
+              plugin: 'rows',
+              state: [
+                {
+                  plugin: 'text',
+                  state: [{ type: 'p', children: [{ text: '' }] }],
+                },
+              ],
+            },
+            interactive: { plugin: 'h5p', state: '' },
+          },
+        },
+      ],
+    },
+  },
+  {
+    ...getInternationalizedPluginStrings(EditorPluginType.Highlight),
+    icon: IconHighlight,
+    type: EditorPluginType.Highlight,
+    initialState: {
+      plugin: 'rows',
+      state: [
+        {
+          plugin: 'highlight',
+          state: { code: '', language: 'text', showLineNumbers: false },
+        },
+      ],
+    },
+  },
+  {
+    ...getInternationalizedPluginStrings(EditorPluginType.Image),
+    icon: IconImage,
+    type: EditorPluginType.Image,
+    initialState: {
+      plugin: 'rows',
+      state: [
+        {
+          plugin: 'image',
+          state: {
+            src: '',
+            caption: {
+              plugin: 'text',
+              state: [{ type: 'p', children: [{ text: '' }] }],
+            },
+          },
+        },
+      ],
+    },
+  },
+  {
+    ...getInternationalizedPluginStrings(EditorPluginType.ImageGallery),
+    icon: undefined,
+    type: EditorPluginType.ImageGallery,
+    initialState: { plugin: 'imageGallery', state: { images: [] } },
+  },
+  {
+    ...getInternationalizedPluginStrings(EditorPluginType.Injection),
+    icon: IconInjection,
+    type: EditorPluginType.Injection,
+    initialState: {
+      plugin: 'rows',
+      state: [{ plugin: 'injection', state: '' }],
+    },
+  },
+  {
+    ...getInternationalizedPluginStrings(EditorPluginType.Multimedia),
+    icon: IconMultimedia,
+    type: EditorPluginType.Multimedia,
+    initialState: {
+      plugin: 'rows',
+      state: [
+        {
+          plugin: 'articleIntroduction',
+          state: {
+            explanation: {
+              plugin: 'text',
+              state: [{ type: 'p', children: [{ text: '' }] }],
+            },
+            multimedia: {
+              plugin: 'image',
+              state: {
+                src: '',
+                caption: {
+                  plugin: 'text',
+                  state: [{ type: 'p', children: [{ text: '' }] }],
+                },
+              },
+            },
+            illustrating: true,
+            width: 50,
+          },
+        },
+      ],
+    },
+  },
+  {
+    ...getInternationalizedPluginStrings(EditorPluginType.SerloTable),
+    icon: IconTable,
```

Comment:
```suggestion
    icon: {
       svgSource: string,
       Component: React.Node
    }
```

What do you think about this?

Reply:
Possibly twice the bundle size, but I'm open to it.

Reply:
I'd say for the React package, we can find a better API. Something like this

```TSX
import React from 'react';
import { EditorPluginType } from '@editor/types/editor-plugin-type';
import * as Icons from './icons'; 

interface IconProps {
  pluginType: EditorPluginType;
  className?: string;
}

export const PluginIcon: React.FC<IconProps> = ({ pluginType, className }) => {
  const SvgIcon = Icons[pluginType as keyof typeof Icons];
  
  if (!SvgIcon) {
    console.warn(`No icon found for plugin type: ${pluginType}`);
    return null;
  }

  return <SvgIcon className={className} />;
};
```

It's a little bit sad, that we have to export the svg sources, but in the future, we can export them exclusively in the editor-web-component. For now, I think we don't have to worry about the bundle size too much and whoever is using the React package, their code bundler should strip out all the unused svgs anyways.